### ISSUE TITLE
[Feature request] API extension: Room language

### ISSUE THREAD

**Issue by mdPlusPlus** [oxen-io/session-pysogs#186]
Hi, maintainer of [sessioncommunities.online](https://github.com/mdPlusPlus/sessioncommunities.online) here.

We're supposing the following API extension:

 - Addition of a language indicator for a given room/community

On [sessioncommunities.online](https://sessioncommunities.online/) we aim to provide users with additional information about available communities. Right now this additional information consists of language flags that indicate what language is spoken in a specific community.  
The goal is to help the user find the right community for them.

At the moment we aquire this information by hand, joining the communities with temporary accounts and relying on online tools to tell us which language we find there. Our current efforts can be seen [here](https://github.com/mdPlusPlus/sessioncommunities.online-languages/blob/main/language_flags.php).  
Ideally operators were able to specify these languages themselves during creation of new communities. That would allow third-party (and first-party, of course) tools to make use of that information programmatically.

There are several possible standards for how to present languages. The most popular ones are probably the [ISO 639 variants](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) with [ISO_639-3](https://en.wikipedia.org/wiki/ISO_639-3) being the most comprehensive one.  
There are other ways of course, using country flags for example. This however isn't ideal, since not every language has a flag that correctly represents it and not every country has a single main language.

As [sessioncommunities.online](https://github.com/mdPlusPlus/sessioncommunities.online) we don't have a clear favorite on what standard to use. It would be great if some discussion about this could take place here, ideally resulting in an addition to the API.

**Comment by KeeJef** [comment]
Sounds good to me, with this and #187 , How do you envision the operator adding this information to a room, would they specify this when they create the room like the room description? I suppose we could expose UI elements in the Session client to specify this information too.

**Comment by mdPlusPlus** [comment]
Yes, like all other information available via the API, I imagine this information to be added using the `sogs` command at creation and being able to be modified later if necessary.
